specification.md

```diff
@@ -1055,10 +1057,168 @@ For example:
 }
 ```
 
-Note that the referenced function definition type in this case must be `rest` (default type).
+Note that the referenced function definition type in this case must be `openapi` (default type).
 
 For more information about functions, reference the [Functions definitions](#Function-Definition) section.
 
+#### Using functions for RESTful Service Invocations
+
+The specification also supports describing REST invocations in the [functions definition](#Function-Definition) using [OpenAPI Paths Object](https://spec.openapis.org/oas/v3.1.0#paths-object).
```

Comment:
I'd like to see here more description on **why** we use openapi for rest here. 
Something to extend that we do not want to create anything proprietary and want to use 
standards like everywhere else. Put some emphasis to users explain them whats the decision behind this.

Reply:
Look at some of the comparisons especially the one we have with google cloud workflow. i think there we mention that they have custom properties for rest / http that we could use to show more of the "why"

Reply:
To me we keep the OpenAPI spec.